### ISSUE TITLE
Feature request: look for bacon.toml in workspace root

### ISSUE THREAD

**Issue by Jasha10** [Canop/bacon#141]
### Feature request:

Look for `bacon.toml` in the workspace root if `bacon.toml` cannot be found in the current package directory.

### Motivation:

If `$PWD` is is a subpackage of a cargo workspace, bacon doesn't look for `bacon.toml` files in the workspace root. For example, if I have the following layout:
```
my_workspace_root
├── bacon.toml
├── Cargo.toml
└── my_package
    ├── Cargo.toml
    └── src
        └── lib.rs
```
If I do the following, bacon won't find the bacon.toml file in `my_workspace_root`:
```
cd my_package
bacon my_custom_job  # fails because bacon doesn't find `../bacon.toml`
```

**Comment by Canop** [comment]
This makes sense.
Just as it makes sense to allow both locations and to have the package's bacon.toml override the workspace's one in the same way it already overrides the global bacon prefs.toml.
I'll look into it.

**Comment by Canop** [comment]
@Jasha10 Can you check https://github.com/Canop/bacon/pull/142 ?

**Comment by Canop** [comment]
Released in 2.10.0
